tests/generation/test_utils.py

```diff
@@ -1102,8 +1115,13 @@ def test_beam_search_low_memory(self):
                 early_stopping=True,
                 low_memory=False,
                 use_cache=True,
+                output_scores=True,
+                output_logits=True,
+                return_dict_in_generate=True,
+                **logits_processor_kwargs,
             )
-            self.assertListEqual(low_output.tolist(), high_output.tolist())
+            # The two outputs must match and their shape must be as expected
+            self._check_similar_generate_outputs(low_output, high_output)
```

Comment:
This is still required, otherwise 1% of chance to fail for `Emu3Vision2TextModelTest`